webnn/resources/test_data/add.json

```diff
@@ -93,691 +93,7 @@
           68.10350799560547,
           -2.3239879608154297
         ],
-        "type": "float32"
-      }
-    },
-    {
-      "name": "add float32 2D tensors",
```

Comment:
Hmm, why are all these other test cases being deleted? Don't we want at least test cases for add that still tests float32?

Reply:
It's just to clearly show test for float16.

Reply:
> The PR title shows that float16 tests are being added
Actually here "add" means add operation.
  
> Don't we want add tests to include both float16 and float32 cases?
Right, adding tests for **float32**，**float16**, and other types of https://webmachinelearning.github.io/webnn/#enumdef-mloperandtype are our target.

Reply:
It's a sample to only show a test for float16.

Reply:
Wait a minute, this is your own private repo (BruceDai/wpt), not web-platform-tests - that was the missing detail! So then I don't even need to sign off since this is your repo anyway, but yeah, it's fine to me.